basepage/views.py

```diff
@@ -33,55 +37,63 @@ def contact(response):
     else:
         form = CreateNewMessage
         if 'submitted' in response.GET:
-           submitted = True
-    return render(response, 'basepage/contact.html', {"form":form, 'submitted':submitted})
+            submitted = True
+    return render(response, 'basepage/contact.html', {"form": form, 'submitted': submitted})
+
 
 @ratelimit(key='ip')
 def films(response):
     return render(response, 'basepage/films.html', {})
 
+
 @ratelimit(key='ip')
 def playing_films(response):
-    movie_list = Movie.objects.all()
-    pgn = Paginator(Movie.objects.all(), 5)
+    min_date = datetime.datetime.now() + datetime.timedelta(days=7)
+    movie_list = ScheduledMovies.objects.filter(end_date__gte=min_date)
+    pgn = Paginator(movie_list, 5)
     page = response.GET.get('page')
     movies = pgn.get_page(page)
 
     return render(response, 'basepage/playing_films.html', {'movie_list': movie_list,
-                                                            'movies':movies})
+                                                            'movies': movies})
```

Comment:
Definitely needs renaming! Those are not movie objects but scheduled movies and this can get very confusing when working with multiple related objects.